bin/jobsub_q

```diff
@@ -1,8 +1,8 @@
-#!/bin/sh
+#!/usr/bin/python3 -I
```

Comment:
This file looks to be almost identical to jobsub_cmd (with the exception of the -global flag and calling condor_q directly).  Is there any reason jobsub_q can't just import jobsub_cmd, add the -global flag, and run that?

Reply:
I've merged the two in the latest push; we do the -global if our command  name is jobsub_q now.